src/schema/v2/me/updateUserInterestMutation.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  GraphQLBoolean,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLUnionType,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { GravityMutationErrorType } from "lib/gravityErrorHandler"
+import { ResolverContext } from "types/graphql"
+import { UserInterest, userInterestType } from "../userInterests"
+
+interface Input {
+  id: string
+  private?: boolean
+}
+
+const SuccessType = new GraphQLObjectType<any, ResolverContext>({
+  name: "UpdateUserInterestSuccess",
+  isTypeOf: (data) => data.id,
+  fields: () => ({
+    userInterest: {
+      type: userInterestType,
+      resolve: async (response) => {
```

Comment:
**question**: is there a need for async?

Reply:
nope, fixed